tsl/test/expected/merge_append_partially_compressed-15.out

```diff
@@ -200,8 +200,8 @@ generate_series(1,3) device;
                      Sort Method: top-N heapsort 
                      ->  Custom Scan (DecompressChunk) on _hyper_1_3_chunk (actual rows=30 loops=1)
                            Filter: (device = ANY ('{1,2,3}'::integer[]))
-                           ->  Index Scan using compress_hyper_2_6_chunk_device__ts_meta_min_1__ts_meta_max_idx on compress_hyper_2_6_chunk (actual rows=3 loops=1)
-                                 Index Cond: (device = ANY ('{1,2,3}'::integer[]))
+                           ->  Seq Scan on compress_hyper_2_6_chunk (actual rows=3 loops=1)
```

Comment:
This seems like a regression? We have a constraint on the first index column so the index should be beneficial

Reply:
For small tables I think it happens often that the Seq Scan is chosen instead of Index Scan. E.g. we often see this change after adding `analyze`.